Packs/qualys/Integrations/Qualysv2/Qualysv2.py

```diff
@@ -3442,16 +3464,15 @@ def main():  # pragma: no cover
             demisto.setLastRun(logs_next_run)
 
         elif command == 'fetch-assets':
-            assets_last_run = demisto.getAssetsLastRun()
-            demisto.debug(f'saved lastrun assets: {assets_last_run}')
-            if skip_fetch_assets(assets_last_run):
```

Comment:
Why did we remove this? 
If we don't need it delete this function - skip_fetch_assets.

Reply:
removed